map/src/drawer/components/favorite/FavoriteGroup.jsx

```diff
@@ -79,36 +79,38 @@ export default function FavoriteGroup({ index, group, enableGroups, setEnableGro
     }
 
     async function addGroupToMap(setProgressVisible, addToMap) {
-        setProgressVisible(true);
-        let newFavoriteFiles = Object.assign({}, ctx.favorites);
-        if (ctx.favorites[group.name]?.url) {
-            newFavoriteFiles[group.name].addToMap = addToMap;
-            ctx.setFavorites({ ...newFavoriteFiles });
-        } else {
-            newFavoriteFiles = await getFavorites(addToMap, newFavoriteFiles);
-            if (addToMap) {
-                let newSelectedGpxFile = {};
-                newSelectedGpxFile.file = Object.assign({}, newFavoriteFiles[group.name]);
-                newSelectedGpxFile.nameGroup = group.name;
-                if (!ctx.selectedGpxFile) {
-                    ctx.selectedGpxFile = {};
-                }
-                if (ctx.selectedGpxFile.markerCurrent) {
-                    newSelectedGpxFile.markerPrev = ctx.selectedGpxFile.markerCurrent;
-                }
-                ctx.setSelectedGpxFile(newSelectedGpxFile);
+        if (FavoritesManager.getGroupSize(group) > 0) {
```

Comment:
Please avoid using OBJECT.something() call because this way is not controlled by the linter.

For external usage it's better to import exact exported function instead of using exported default object.

For local usage (in this module) it's better to call direct function.